shared/actions/config/index.js

```diff
@@ -422,7 +448,7 @@ function* configSaga(): Saga.SagaGenerator<any, any> {
   )
 
   // Like handshake but in reverse, ask sagas to do stuff before we tell the server to log us out
-  yield* Saga.chainAction<ConfigGen.LogoutPayload>(ConfigGen.logout, startLogoutHandshake)
+  yield* Saga.chainAction<ConfigGen.LogoutPayload>(ConfigGen.logout, startLogoutHandshakeIfAllowed)
```

Comment:
@chrisnojima does going through this step before starting the logout handshake look ok to you?

Reply:
👍